pre commit setup (#99)
avantifellows/plio-frontend: README.md

# Plio Frontend
[![License: MIT](https://img.shields.io/github/license/avantifellows/plio-frontend?color=blue&style=flat-square)](LICENSE)
![GitHub issues](https://img.shields.io/github/issues-raw/avantifellows/plio-frontend?style=flat-square)
![Pre-commit checks](https://img.shields.io/github/workflow/status/avantifellows/plio-frontend/pre-commit/master?label=Pre-commit%20checks&style=flat-square)
[![Discord](https://img.shields.io/discord/717975833226248303.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2&style=flat-square)](https://discord.gg/29qYD7fZtZ)

> Plio enables educators to teach better through videos. Educators can overlay questions on top of any video to make learning interactive, gather detailed insights on student engagement and receive feedback on their content itself. 

Reviewer: ```suggestion
Plio enables educators to teach better through videos. Educators can overlay questions on top of any video to make learning interactive, gather detailed insights on the engagement of learners and receive feedback on their content itself. 
```

# Plio Frontend

> [![License: MIT](https://img.shields.io/github/license/avantifellows/plio-frontend?color=blue&style=flat-square)](LICENSE)

Reviewer: I'll make some changes to this README in a commit.

![GitHub issues](https://img.shields.io/github/issues-raw/avantifellows/plio-frontend?style=flat-square)
![Pre-commit checks](https://img.shields.io/github/workflow/status/avantifellows/plio-frontend/pre-commit/master?label=Pre-commit%20checks&style=flat-square)
[![Discord](https://img.shields.io/discord/717975833226248303.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2&style=flat-square)](https://discord.gg/29qYD7fZtZ)

> Plio is an open-source AI-based platform that enables personalized learning through free interactive videos.

Reviewer: ```suggestion
Plio enables educators to teach better through videos. Educators can overlay questions on top of any video to make learning interactive, gather detailed insights on student engagement and receive feedback on their content itself. 
```

> This repo contains code for the frontend, in Vue.

Reviewer: ```suggestion
This repo contains code for the frontend, in VueJS.
```

> It includes features for:

Reviewer: ```suggestion
```

> 1. A player that overlays items (currently assumed to be questions) on top of any video (currently assumed to be from YouTube(
> 2. Recording the phone number of the user (currently without any authentication).

Reviewer: ```suggestion
```

> ## Priorities

Reviewer: ```suggestion
```

> We are keeping the features *extremely* simple, because our target market is students from low-income backgrounds in India.

Reviewer: ```suggestion
```

> Some of them don't have their own phones and borrow from their families, and some of them have only KaiOS feature phones.

Reviewer: ```suggestion
```

> Hence, we have decided that the platform will:

Reviewer: ```suggestion
```

> 1. Recommend high-quality free video content in local languages,

Reviewer: ```suggestion
```

> 2. Automatically build interactive lessons on top of these videos,

Reviewer: ```suggestion
```

> 3. Work on feature phones,

Reviewer: ```suggestion
```

> 4. Be freely available.

Reviewer: ```suggestion
```

## Installation
To set up this project, visit the [installation steps](docs/INSTALLATION.md).

## Deployment
Visit our [deployment guideline](docs/DEPLOYMENT.md) for setting up this repository on a production environment.

> ##  Testing

Reviewer: ```suggestion
```

> Our testing is powered by the open source plan of [BrowserStack](https://www.browserstack.com/)

Reviewer: ```suggestion
```

![Pre-commit checks](https://img.shields.io/github/workflow/status/avantifellows/plio-frontend/pre-commit/master?label=Pre-commit%20checks&style=flat-square)
[![Discord](https://img.shields.io/discord/717975833226248303.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2&style=flat-square)](https://discord.gg/29qYD7fZtZ)

> Plio enables educators to teach better through videos. Educators can overlay questions on top of any video to make learning interactive, gather detailed insights on student engagement and receive feedback on their content itself. 

Reviewer 2: We should replace `student` here with `learner` somehow right?
